packages/salesforcedx-apex-debugger/package.json

```diff
@@ -45,8 +45,8 @@
     "watch": "tsc -watch -p .",
     "clean": "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
     "test": "npm run test:unit && npm run test:integration",
-    "test:unit": "./node_modules/.bin/cross-env VSCODE_NLS_CONFIG={} ./node_modules/.bin/nyc ./node_modules/.bin/_mocha --recursive out/test/unit --reporter mocha-multi-reporters --reporter-options configFile=../../config/mochaUnitTestsConfig.json",
-    "test:integration": "./node_modules/.bin/cross-env VSCODE_NLS_CONFIG={} ./node_modules/.bin/nyc ./node_modules/.bin/_mocha --recursive out/test/integration --reporter mocha-multi-reporters --reporter-options configFile=../../config/mochaIntegrationTestsConfig.json"
+    "test:unit": "node ./node_modules/nyc/bin/nyc.js ./node_modules/mocha/bin/_mocha --recursive out/test/unit --exit --reporter mocha-multi-reporters --reporter-options configFile=../../config/mochaUnitTestsConfig.json",
```

Comment:
Had to update to using the direct module references since CircleCI Windows was not properly able to reference the dependencies when using `node_modules/.bin/...`

Reply:
I do not know enough about this piece - do you know why this part isn't needed for test:unit? `./node_modules/cross-env/dist/bin/cross-env.js VSCODE_NLS_CONFIG={}`. I know VSCODE_NLS_CONFIG is used for localization

Reply:
Yes, it's because when we run integration tests we spin up a local instance of vscode that we later use to run the tests on. We need it since those tests have direct references to vscode apis, etc. For unit tests we do not do that cause we don't have any dependencies with vscode at the test level.